scripts/init-submodules-no-riscv-tools.sh

```diff
@@ -11,14 +11,14 @@ scripts_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 # ignore riscv-tools for submodule init recursive
 # you must do this globally (otherwise riscv-tools deep
 # in the submodule tree will get pulled anyway
-git config --global submodule.riscv-tools.update none
+git config --global submodule.toolchains/riscv-tools.update none
 git config --global submodule.esp-tools.update none
```

Comment:
nit: see my other nit on "path to git submodules"

Reply:
done